_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Introducing open source Kubernetes Operator for OpenSearch"
+authors:
+  - Opster
```

Comment:
Please change this to the individuals who wrote the post instead of the company name and include their picture instead of Opster